webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala

```diff
@@ -334,20 +336,27 @@ class ComplianceAPIService(
       _             <- TimingDebugLoggerPure.trace(s"getByRulesCompliance - findRuleNodeStatusReports in ${t6 - t5} ms")
 
     } yield {
+      println(s"p0000000000000000000000000000000000 $directives")
+      val overridesByRule = reportsByNode.toList.flatMap(report => report._2.overrides.groupBy(_.policy.ruleId)).toMap
+      val overridesRules = overridesByRule.view.mapValues(_ => AggregatedStatusReport(Nil)).toMap
 
+      println(s"overridesByRule.toList ${overridesByRule.toList}")
+      println(s"overridesRules ${overridesRules.toList.foreach(println)}")
       val reportsByRule = reportsByNode.flatMap { case (_, status) => status.reports }.groupBy(_.ruleId)
+      val reportsByRule2 = reportsByNode.flatMap { case (_, status) => status.byRules }
       val t7            = System.currentTimeMillis()
       TimingDebugLoggerPure.logEffect.trace(s"getByRulesCompliance - group reports by rules in ${t7 - t6} ms")
-
+      val finalReports = overridesRules ++ reportsByRule2
```

Comment:
I was trying to send the overridden (skipped) directives to send it to the `getRulesComplianceId` API, but it seems like is not working, we don't get the overridden directives in the response